dml/import_schools.sql

```diff
@@ -39,6 +40,7 @@ insert into schools (
 
 select
 	sr."URN"::integer,
+	sr."UKPRN",
```

Comment:
This _might_ be required, feel free to ignore if it's not though:

```suggestion
	sr."UKPRN"::integer,
```

Reply:
I've added — the [website](https://www.ukrlp.co.uk/#:~:text=UK%20Provider%20Reference%20Number%20(UKPRN,deactivated%20providers%20in%20the%20search%3F) states that the UKPRN is an integer always beginning with `1`, so no need to deal with stray `0`s at the start of the code